.github/workflows/fuzzers.yml

```diff
@@ -33,13 +33,7 @@ jobs:
           - aflplusplus_qemu
           - honggfuzz_qemu
           # temporary variants.
```

Comment:
Is aflplusplus_havoc a temporary variant, if not this comment can be removed.

Reply:
yes that is a temporary variant (changed the havoc iteration algo, the previous attempt was not good, hopefully this one performs better)